library/network_connections.py

```diff
@@ -772,6 +777,9 @@ def connection_compare(
         if compare_flags is None:
             compare_flags = NM.SettingCompareFlags.IGNORE_TIMESTAMP
 
+        # this is really odd, but I'm assuming there is a double-negative
+        # here for a reason . . . .
+        # pylint: disable=unneeded-not
         return not (not (con_a.compare(con_b, compare_flags)))
```

Comment:
@thom311 can you help here? Do you remember why we have this?

Reply:
yes, it can be removed.

Reply:
Does that mean I can change this to 
```
   return con_a.compare(con_b, compare_flags)
```
?

Reply:
Done.